.github/workflows/deploy.yml

```
@@ -14,12 +14,13 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-go@v4
       with:
-        go-version: '~1.20.6'
+        go-version: '1.20'
     - name: deps
       run: go mod download
     - name: build
       run: |
         GOARCH=arm64 make module.tar.gz && mv module.tar.gz module-arm64.tar.gz
+        rm wifi # otherwise make will not recreate it
         GOARCH=amd64 make module.tar.gz && mv module.tar.gz module-amd64.tar.gz
```

[opt] this feels slightly cleaner:

```suggestion
        GOARCH=arm64 make --always-make module.tar.gz && mv module.tar.gz module-arm64.tar.gz
        GOARCH=amd64 make --always-make module.tar.gz && mv module.tar.gz module-amd64.tar.gz
```

alternatively it might be worth adding these steps as targets to the Makefile to have less logic in the workflow